FuzzTesting/do_build.sh

```diff
@@ -6,10 +6,47 @@ readonly FuzzTestingDir=$(dirname "$(echo $0 | sed -e "s,^\([^/]\),$(pwd)/\1,")"
 
 cd "${FuzzTestingDir}"
 
+# Default to both
+CMD_CONFIGS=("debug" "release")
+
+while [[ $# != 0 ]]; do
+  case "${1}" in
+    --debug )
+      CMD_CONFIGS=("debug")
+      ;;
+    --release )
+      CMD_CONFIGS=("release")
+      ;;
+    --both )
+      CMD_CONFIGS=("debug" "release")
+      ;;
+    -*)
+      echo "ERROR: Unknown option: ${1}" 1>&2
+      exit 1
+      ;;
+    *)
+      echo "ERROR: Unknown argument: ${1}" 1>&2
+      exit 1
+      ;;
+  esac
+  shift
+done
+
+declare -a CMD_BASE
 if [ "$(uname)" == "Darwin" ]; then
-  xcrun \
-    --toolchain swift \
-    swift build -c debug -Xswiftc -sanitize=fuzzer,address -Xswiftc -parse-as-library
+  CMD_BASE=(
+    xcrun
+      --toolchain swift
+      swift build -Xswiftc -sanitize=fuzzer,address -Xswiftc -parse-as-library
+  )
 else
-  swift build -c debug -Xswiftc -sanitize=fuzzer,address -Xswiftc -parse-as-library
+  CMD_BASE=(
+    swift build -Xswiftc -sanitize=fuzzer,address -Xswiftc -parse-as-library
+  )
 fi
+
+for CMD_CONFIG in "${CMD_CONFIGS[@]}"; do
+  echo "Building: ${CMD_CONFIG}"
+  echo "${CMD_BASE[@]}" -c "${CMD_CONFIG}"
```

Comment:
I believe the `[@]` notation is a bash-ism, in case that matters here.

Reply:
It is, but the file has `#!/bin/bash` at the top, so we're good.